content/applications/inventory_and_mrp/inventory/shipping_receiving/picking_methods/batch.rst

```diff
@@ -125,14 +116,46 @@ Doing so opens an :guilabel:`Add to batch` pop-up window, wherein the employee
 Choose from the two radio options to add to :guilabel:`an existing batch transfer` or create
 :guilabel:`a new batch transfer`.
 
+Add a :guilabel:`Description` for this batch.
+
+.. tip::
+   The :guilabel:`Description` field can be used to add additional information to help workers
+   identify the source of the batch, where to place the batch, what shipping containers to use, etc.
+
 To begin with a draft, select the :guilabel:`Draft` checkbox.
 
 Conclude the process by clicking :guilabel:`Confirm`.
 
 .. image:: batch/add-to-batch-window.png
-   :align: center
    :alt: Show *Add to batch* window to create a batch transfer.
 
+Automatic batches
+-----------------
+
+Batches can be automatically created and assigned based on several criteria. The *Automatic Batches*
+option is defined on the *operation type* level, which enables the creation of batches with distinct
+grouping criteria for each operation type.
```

Comment:
I think the readability of this explanation could be made more clear. Not sure how I would do it exactly, but I think what's sticking out to me is the circular logic of "...operation type level, which enables creation...for each operation type (level)"